index.css

```diff
@@ -0,0 +1,569 @@
+/* @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100&family=Roboto:wght@300&display=swap'); */
+@media only screen and (min-width: 892px) {
+  .nav {
+    overflow: hidden;
+    background-color: #3e3c3c;
+    width: 1440px;
+    height: 72px;
+  }
+
+  .nav a {
+    overflow: hidden;
+    float: left;
+    display: block;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    position: relative;
+    left: 70%;
+    top: 10px;
+    width: 59px;
+    height: 20px;
+  }
+
+  .nav a:hover {
+    background-color: #fff5e1;
+  }
+
+  .image1 {
+    overflow: hidden;
+    position: relative;
+    left: 70%;
+    top: 22px;
+  }
+
+  .image1:hover {
+    background-color: aquamarine;
+  }
+
+  .nombre {
+    position: relative;
+    left: 12%;
+    color: #fff;
+    bottom: 20px;
+    width: 92px;
+    height: 20px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .main {
+    background-color: #1c1a19;
+    width: 1440px;
+    height: 870px;
+    position: relative;
+    bottom: 67px;
+  }
```

Comment:
- [ ] The mobile version of the page still looks really disorganized. 

I think the reason behind this is the usage of the `position: relative`. You are positioning an element relative to its parent. So if the position of that parent element changes (for instance via a browser resize) then so too will elements positioned in this way. 
This is making elements go out of natural document flow and making the page less responsive. It is really hard to make positioned elements responsive. The result always will be different according to users' system specifications like screen size, browser version, etc... 

It will be more manageable to create responsive layouts if you try to position your elements with the use of more responsive friendly CSS elements like margin, padding, flexbox, grid, etc...

![Screenshot 2022-02-28 at 11-32-52 Microwebpage](https://user-images.githubusercontent.com/1642014/155953017-63e4f494-5b10-44a3-9c35-6f90c60fa97c.png)